mach_eval/analyzers/electromagnetic/stator_wdg_res.py

```diff
@@ -60,20 +65,31 @@ def analyze(self, problem: StatorWindingResistanceProblem):
         Kov = problem.Kov
         sigma_cond = problem.sigma_cond
         slot_area = problem.slot_area
+        n_layers = problem.n_layers
 
         # Length between adjacent slots evaluated at median depth of slot
         tau_u = 2 * np.pi / Q * (r_si + d_sp + d_st / 2)
-        # Length of end winding (one side of coil)
-        l_end_wdg = 0.5 * np.pi * (tau_u + w_st) / 2 + tau_u * Kov * (y - 1)
-        # Length of end winding (both sides of coil)
-        l_coil_end_wdg = 2 * l_end_wdg
+        # Length of end winding
+        l_ew = 0.5 * np.pi * (tau_u + w_st) / 2 + tau_u * Kov * (y - 1)
```

Comment:
Let's split this `l_ew` calculation up into the `l1` and `l2` calculations described in the RST file  (to get better alignment in approach between the RST and PY files)